approval_polls/templates/base.html

```diff
@@ -9,10 +9,26 @@
     <link rel='stylesheet' type='text/css' href='{% static 'approval_polls/style.css' %}'>
     <script src='{% static 'vendor/jquery-1.11.3.min.js' %}'></script>
     <script src='{% static 'vendor/bootstrap/js/bootstrap.min.js' %}'></script>
+    <script src='{% static 'vendor/jstz/js/jstz.min.js' %}'></script>
     <title>Approval Polls from The Center for Election Science</title>
     {% block head %}{% endblock %}
   </head>
   <body>
+    {% load tz %}
+    {% get_current_timezone as TIME_ZONE %}
+    <script>
+      var timezone = jstz.determine();
```

Comment:
It will be good to have a short comment explaining why we need to do this. 
Also, since we would usually prefer to separate the JS code from the HTML in its own file, but in this case we make an exception since we need to access template variables in the JS code.